test/ibmq/test_ibmq_job.py

```diff
@@ -233,9 +249,20 @@ def test_get_jobs_from_backend(self, provider):
         self.log.info('time to get job statuses: %0.3f s',
                       time.time() - start_time)
 
+    def test_retrieve_job_backend(self, provider):
+        """Test retrieving a single job from a backend."""
+        backend = provider.get_backend('ibmq_qasm_simulator')
+        qobj = assemble(transpile(self._qc, backend=backend), backend=backend)
+        job = backend.run(qobj)
+
+        rjob = backend.retrieve_job(job.job_id())
```

Comment:
Also for cleaning up - can you rename to `retrieved_job` or something a bit more descriptive?